server/tests/model.specs/articles.specs.js

```diff
@@ -0,0 +1,47 @@
+import mocha from "mocha";
+import chai from "chai";
+import database from "../../models/index";
+import seeds from "./mode.fixtures";
+
+const sequelize = database.sequelize;
+const { expect } = chai;
+const Article = database["Articles"];
+const articleSeed = seeds.article;
+describe("Articles", () => {
+  it("should create an instance of Articles", async () => {
+    const article = await Article.create(articleSeed);
+    const {
+      title,
+      authorId,
+      categoryId,
+      body,
+      description,
+      favouriteCount,
+      numberOfReads
+    } = article.dataValues;
+    expect(article).to.be.an.instanceOf(Article);
+    expect(article).to.have.property("id");
+    expect(articleSeed.title).to.equal(title);
+    expect(articleSeed.body).to.equal(body);
+    expect(articleSeed.description).to.equal(description);
+    expect(articleSeed.categoryId).to.equal(categoryId);
+    expect(favouriteCount).to.equal(0);
+    expect(numberOfReads).to.equal(0);
```

Comment:
These are too many tests in one it block, please find a way to break them.

Reply:
Fixed